README.md

```diff
@@ -90,25 +90,22 @@ The relayer binary accepts a path to a JSON configuration file as the sole argum
 
 The relayer is configured via a JSON file, the path to which is passed in via the `--config-file` command line argument. The following configuration options are available:
 
-`"log-level": "debug" | "info" | "warn" | "error" | "fatal" | "panic"`
+`"log-level": "verbo" | "debug" | "info" | "warn" | "error" | "fatal" | "panic"`
 
 - The log level for the relayer. Defaults to `info`.
 
-`"network-id": unsigned integer`
-
-- The ID of the Avalanche network to which the relayer will connect. Defaults to `1` (Mainnet).
-
 `"p-chain-api-url": string`
 
 - The URL of the Avalanche P-Chain API node to which the relayer will connect. This API node needs to have the following methods enabled:
-  - info.peers
   - platform.getHeight
   - platform.validatedBy
   - platform.getValidatorsAt
 
-`"encrypt-connection": boolean`
+`"info-api-url": string`
```

Comment:
i'm not too familiar with the info api nodes, is it common for the p chain api node to have info api enabled as well?

Reply:
thinking if worth have more documentation about how to find their info api node, or whether to add a comment about testing with the p chain api node

Reply:
It's entirely up to the node operator which APIs to support. The P-Chain and Info APIs are independent, so we shouldn't assume that they'll both be enabled on the same node here.

Reply:
The public APIs support both the info endpoints and most of the P-chain endpoints (with the exception of `getValidatorsAt`. 

```
$ curl 'https://api.avax-test.network/ext/info' \
> --header 'Content-Type: application/json' \
> --data '{
>     "jsonrpc":"2.0",
>     "id"     :1,
>     "method" :"info.getNetworkID"
> }'
{"jsonrpc":"2.0","result":{"networkID":"5"},"id":1}
```

I probably would have gone the route of having a single `avalancheAPIURL` configuration option and adding the path extensions as needed in the application, but not opposed to having these split up in the event that the node used for the P-chain API (possibly private node) doesn't have the Info API enabled (all public APIs have the Info API available).

Reply:
Yeah, in the case of the public API, both the info and P-chain API share the same base URL. That's not true in general though, so I think it's better to make as few assumptions here as we can.

Reply:
Yeah, totally agree. Thanks for elaborating.

Reply:
is there any additional comment/documentation we can add for finding the info api, or is this relatively understood/straightforward for node operators?

Reply:
makes sense, I agree